examples/coexist_klog_v1_using_logr/coexist_klog_v1.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"github.com/go-logr/logr"
+
+	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
+)
+
+type klogLogger struct{}
+
+func (kw klogLogger) Info(msg string, keysAndValues ...interface{}) {
+	klog.InfoDepth(5, msg)
```

Comment:
where does 5 come from? if this is based on internal routing within klog, we shouldn't recommend this (it will break on klog version updates)

Reply:
same as above. I can add tests for both to fix the depth. will that help?